File: Extract_Method.py
# After
def printOwing(self):
    self.printBanner()

    # print details
    print("name:", self.name)
    print("amount:", self.getOutstanding())

# Refactoring
def printOwing(self):
    self.printBanner()
    self.printDetails(self.getOutstanding())

def printDetails(self, outstanding):
    print("name:", self.name)
    print("amount:", outstanding)

File: Extract_Method.js
// After
function printOwing(self) {
  self.printBanner();

  // print details
  console.log("name: ", self.name);
  console.log("amount: ", self.getOutstanding());
}    

// Refactoring
function printOwing(self) {
  self.printBanner();
  self.printDetails(self.getOutstanding());
}    

function printDetails(self, outstanding) {
  console.log("name: ", self.name);
  console.log("amount: ", outstanding);
}
